<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Решение систем линейных уравнений</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QComboBox" name="combobox_method">
         <item>
          <property name="text">
           <string>Метод Гаусса </string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Метод Крамера</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Метод Зейделя</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Метод верхней релаксации</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Метод простых итераций</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>LU-разложение</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushbutton_solve">
         <property name="text">
          <string>Решить выбранным методом</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushbutton_solve_all">
         <property name="text">
          <string>Решить всеми методами</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushbutton_toggle">
         <property name="text">
          <string>Скрыть последнее решение</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="vlayout_system">
         <item>
          <widget class="QLabel" name="label_system">
           <property name="text">
            <string>Ax=b: введите значения матрицы A и вектора b ниже</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="table_system"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="vlayout_solution">
         <item>
          <widget class="QLabel" name="label_solution">
           <property name="text">
            <string>Решение системы (epsilon = 0.0001)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="table_solution">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_fastest_method">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Самый быстрый метод не определен</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>812</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_start"/>
  </widget>
  <action name="action_start">
   <property name="text">
    <string>Начать</string>
   </property>
   <property name="toolTip">
    <string>Start</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
